src/main/java/net/minecraftforge/common/ToolActions.java

```diff
@@ -80,7 +85,22 @@
      *  @see {@link IForgeItem#getSweepHitBox}
      */
     public static final ToolAction SWORD_SWEEP = ToolAction.get("sword_sweep");
-    
+
+    /**
+     *  Exposed by shears to allow harvesting tool behaviours (example: harvesting beehives)
+     */
+    public static final ToolAction SHEARS_HARVEST = ToolAction.get("shears_harvest");
+
```

Comment:
This is a little confusing, you have this named harvest but use 'dig' in all the loot tables.
These need better docs or something to describe that is what

Reply:
The "dig" action is used for breaking blocks. The "harvest" action is used for blocks with a right click behaviour that allows a harvest operation for shears (like beehives with honey level 5)
I will extend the desciption. :)

Reply:
I hope it's a little clearer now :)

Reply:
Maybe we should be more specific and say "right-click harvesting" instead of just "harvesting"

Reply:
> This action is exposed by shears and corresponds to a harvest action that is triggered with a right click on a block that supports such behaviour. Example: Right click with shears on a beehive with honey level 5 to harvest it

This is the new text. Or do you mean to rername the action?

Reply:
Oh, ignore me. I was looking idly while working and didn't notice I was seeing the *old* code.